docs/src/modules/javascript/pages/kickstart.adoc

```diff
@@ -116,6 +116,21 @@ You should now see the following source files in your project:
 
 NOTE: The `lib/generated` directory contains files used by the tools. Any changes made to files in this folder are overwritten on the next build.
 
+=== Unit and integration tests
+Akka Serverless codegen creates a unit test stub for each class within `counter.test.js`. Use these stubs as a starting point to test the logic in your implementation. You can use any of the common Node.js testing tools, such as Mocha or Jest.
+
+[source,javascript]
+----
+include::../../../../../samples/valueentity-counter/test/counter.test.js[]
```

Comment:
It's not possible to include sources outside of Antora's source tree. This sample gets dynamically copied under examples, so it can be referenced with:

```suggestion
include::example$valueentity-counter/test/counter.test.js[]
```